bin/drone/destroy-errored-agents

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -e
+
+# This script will terminate all drone agents that are in 'error' state. This
+# happens occasionally, and we have not been able to pin down the exact reason
+# why. This was created when we were using the following versions, and may not
+# be needed if future bugfixes resolve this issue.
+#
+# - drone/autoscaler:1.8.2
+# - drone/drone:1.9.0
+#
+# If it's too fancy for you, you can always run the following command.
+#
+# ```
+# drone server ls -l | grep error | awk '{print $1}' | xargs -tL1 drone server destroy`
+# ```
+
+# Check if DRONE_TOKEN is valid
+output=$(drone server ls -l 2>&1)
+if [[ "$output" == *"Error: you must provide your Drone access token."* ]]; then
+    echo "DRONE_TOKEN is either not set or invalid. Follow instructions at https://drone.cdn-code.org/account to set your token."
```

Comment:
That URL doesn't have any instructions for me, just the token value itself; instructions are in the google doc

Reply:
Should we also warn about `DRONE_SERVER` and/or `DRONE_AUTOSCALER`, too? Or perhaps just have logic in here to set those values?

Reply:
Good point. I have those in my bashrc so I didn't think about them.

Reply:
Updated the error output to consider all three potentially missing. I don't want to hardcode the domain names here because that's potentially worth keeping out of the public repo.

Reply:
scratch that, the domains are already very visible in the repo. so I'm going to hardcode those two variables.